tasks-by-users/samar4tech/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  min-height: 100vh;
+  background: #333;
```

Comment:
```suggestion
  background-color: #333;
```

Reply:
i did the same in my 1st project( as you suggested here) but there Rahmatullah bhai suggested me to use only bg rather than bg-color.
Which one should i need to opt?

Reply:
Using `background-color` is a good approach but in this particular file you have used 2 background properties at the same time, that's the reason why Abbas Bhai suggested this changes @samar4tech